desktop/src/main/java/bisq/desktop/main/dao/bonding/reputation/MyReputationView.java

```diff
@@ -312,6 +312,9 @@ public void updateItem(final MyReputationListItem item, boolean empty) {
             }
         });
         tableView.getColumns().add(column);
+        column.setComparator(Comparator.comparing(MyReputationListItem::getAmount));
+        column.setSortType(TableColumn.SortType.ASCENDING);
+        tableView.getSortOrder().add(column);
```

Comment:
Again pointless defining a string to be sorted for as it is the default behavior.